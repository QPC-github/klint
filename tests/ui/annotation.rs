#![crate_type = "lib"]

#[klint::preempt_count]
fn a() {}

#[klint::preempt_count()]
fn b() {}

#[klint::preempt_count(adjust = )]
fn c() {}

#[klint::preempt_count(expect = )]
fn d() {}

#[klint::preempt_count(expect = ..)]
fn e() {}

#[klint::preempt_count(unchecked)]
fn f() {}
