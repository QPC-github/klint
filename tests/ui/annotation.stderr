error: incorrect usage of `#[kint::preempt_count]`
 --> $DIR/annotation.rs:6:23
  |
6 | #[klint::preempt_count()]
  |                       ^^
  |
  = help: at least one of `adjust` or `expect` property must be specified
  = note: `#[forbid(klint::incorrect_attribute)]` on by default

error: incorrect usage of `#[kint::preempt_count]`
 --> $DIR/annotation.rs:9:33
  |
9 | #[klint::preempt_count(adjust = )]
  |                                 ^
  |
  = help: an integer expected

error: incorrect usage of `#[kint::preempt_count]`
 --> $DIR/annotation.rs:3:1
  |
3 | #[klint::preempt_count]
  | ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: correct usage looks like `#[kint::preempt_count(...)]`

error: incorrect usage of `#[kint::preempt_count]`
  --> $DIR/annotation.rs:12:33
   |
12 | #[klint::preempt_count(expect = )]
   |                                 ^
   |
   = help: a range expected

error: incorrect usage of `#[kint::preempt_count]`
  --> $DIR/annotation.rs:18:23
   |
18 | #[klint::preempt_count(unchecked)]
   |                       ^^^^^^^^^^^
   |
   = help: at least one of `adjust` or `expect` property must be specified

error: aborting due to 5 previous errors

